data-imports/clean_utils.py

```diff
@@ -81,7 +82,7 @@ def add_boroid(table, column):
         """.format(table=lookup_table)
     if not column_exists(table, 'bbl'):
         sql = sql + """
-        ALTER TABLE {table} ADD COLUMN {columnid} bigint(13) NOT NULL DEFAULT 0;
+        ALTER TABLE {table} ADD COLUMN {columnid} bigint(13) NULL DEFAULT NULL;
```

Comment:
Hmm, there is actually an error here, but not in this line: in line 83, it should be "column_exists(table, columnid)".